src/Trakx.IndiceManager.Server.Tests/Data/CoinbaseTransactionHandlerTest.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using Microsoft.Reactive.Testing;
+using Xunit;
+using Trakx.IndiceManager.Server.Data;
+using System.Reactive.Concurrency;
+using FluentAssertions;
+using Trakx.Coinbase.Custody.Client.Models;
+using Xunit.Abstractions;
+
+namespace Trakx.IndiceManager.Server.Tests.Data
+{
+    public class CoinbaseTransactionHandlerTest
+    {
+        private readonly CoinbaseTransactionsHandler _coinbaseTransactionsHandler;
+
+        public CoinbaseTransactionHandlerTest(ITestOutputHelper output)
+        {
+            var logger = output.ToLogger<CoinbaseTransactionsHandler>();
+            _coinbaseTransactionsHandler = new CoinbaseTransactionsHandler(logger);
+        }
+
+        [Fact]
+        public void CoinbaseTransactionHandler_should_throws_notification_when_new_transaction_published_on_stream()
```

Comment:
```suggestion
        public void CoinbaseTransactionHandler_should_send_notification_when_new_transaction_published_on_stream()
```
throw~s~ 😂 throw immediately makes me think of exception...